src/OMSimulatorLib/TLMBusConnector.cpp

```diff
@@ -190,8 +235,8 @@ void oms3::TLMBusConnector::sortConnectors()
 
 oms_status_enu_t oms3::TLMBusConnector::registerToSockets(TLMPlugin *plugin)
 {
-  if(sortedConnectors.empty())
-    return logError("All required connectors not added to TLM bus");
+  if(sortedConnectors.empty() && (getActualBus() == this))
+    return logError("No connectors in TLM bus: "+std::string(parentSystem->getFullCref())+"."+std::string(getName()));
```

Comment:
It would be good to move also this error message to `Logging.h`.

Reply:
Why? I thought the predefined messages in Logging.h was for reusing common messages. This one will likely never be used in any other location.